aci/data_source_aci_pimifpol.go

```diff
@@ -0,0 +1,83 @@
+package aci
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/ciscoecosystem/aci-go-client/v2/client"
+	"github.com/ciscoecosystem/aci-go-client/v2/models"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func dataSourceAciPIMInterfacePolicy() *schema.Resource {
+	return &schema.Resource{
+		ReadContext:   dataSourceAciPIMInterfacePolicyRead,
+		SchemaVersion: 1,
+		Schema: AppendBaseAttrSchema(AppendNameAliasAttrSchema(map[string]*schema.Schema{
+			"tenant_dn": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"auth_key": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"auth_t": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"ctrl": {
+				Type:     schema.TypeList,
+				Computed: true,
+				Elem:     &schema.Schema{Type: schema.TypeString},
+			},
+			"dr_delay": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"dr_prio": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"hello_itvl": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"jp_interval": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"name": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"secure_auth_key": {
```

Comment:
What is the difference between secure auth_key and auth_key?

Reply:
Are those returned from the API?

Reply:
They are not returned from the API and they were not configurable either. In the UI secure_auth_key becomes available to the user when auth_type is changed to "ah-md5". I'm not sure about auth_key.

Reply:
I removed them from the resource